### PR TITLE
Cell blueprints now use the /empty variants

### PR DESCRIPTION
**Pull request by Cyberboss**
:cl:
balance: Printed power cells must now be charged before use
/:cl:

This is a bad power balance meme that needs to go

### CODE REVIEW

**Comment by arsserpentarium** [comment]
This way cells needs to be charged faster.

**Comment by davethwave** [comment]
Nice to see this happening. There were more than a few times where I was tempted to just build a bluespace battery SMES to become power independent the rest of the round.

File: code/modules/research/designs/power_designs.dm [hunk 1]
```diff
@@ -20,7 +20,7 @@
 	build_type = PROTOLATHE | AUTOLATHE | MECHFAB
 	materials = list(MAT_METAL = 700, MAT_GLASS = 60)
 	construction_time=100
-	build_path = /obj/item/stock_parts/cell/high
+	build_path = /obj/item/stock_parts/cell/high/empty
```
Comment (ShizCalev) [inline code/modules/research/designs/power_designs.dm]: how does this compile

**Review by ShizCalev — APPROVED** [review]

**Comment by PKPenguin321** [comment]
Larger cells take an age to recharge

**Comment by coiax** [comment]
I tried to do this before. I support this idea.

**Comment by DaedalusGame** [comment]
I don't support this, instead producing a cell should take power from the APC

**Comment by PKPenguin321** [comment]
well that would be dumb because then you cant print a larger cell type than whatever cell type already exists in the room APC

**Comment by DaedalusGame** [comment]
There is clearly only one solution then

Lathes need a wirenode underneath.

**Comment by Tlaltecuhtli** [comment]
whats the point of printing the cell if its empty

**Comment by Kromgar** [comment]
make chargers faster then

**Comment by Kromgar** [comment]
Motherfucker.


This is gonna be fun on bagil where we constantly have to resort to inducers to power cargo

**Comment by BlueNothing** [comment]
Cells will take an extended period of time to charge, and in this arrangement, there's a strong *dis*incentive to upgrade cells, because of the outage period, the charge period, and the, for the most part, failure of cell upgrades to allow independence from the grid. I do not support this arrangement, and think that cells starting powered is reasonable, and incentivizes individual and departmental actions to handle power problems in the event that engineering is AWOL, or worse, incompetent, as they often appear to be.

**Comment by davethwave** [comment]
https://github.com/tgstation/tgstation/pull/35517#issuecomment-364704503

**Comment by Ausops** [comment]
>bad power balance meme
>doesn't state why

**Pull request merged**